src/styles/glabal.js

```diff
@@ -0,0 +1,96 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  vertical-align: baseline;
+  text-decoration: none;
+  outline: none;
+  box-sizing: border-box;
+  background-color: transparent;
+}
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+a,
+a:link,
+a:visited,
+a:active {
+  text-decoration: none;
+  color: unset;
+}
+ol,
+ul,
+li {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+button {
+  background: none;
+  cursor: pointer;
+}
+input[type='button'],
+input[type='submit'],
+input[type='reset'],
+input[type='radio'],
+label {
+  cursor: pointer;
+}
+input[type='radio'] {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+input[type='number'] {
+  -moz-appearance: textfield;
+}
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+  
+  body {
+    @supports (height: 100svh) { height: 100svh; }
+    @supports not (height: 100svh) { height: 100vh; }
+
+    @media (max-width: 768px) {
+      @supports (width: 100svw) { width: 100svw; }
+      @supports not (width: 100svw) { width: 100vw; }
+    }
+
+    @media (min-width: 768px) {
+      width: 768px;
+      margin: 0 auto;
+    }
```

Comment:
❗ 데스크탑 기준으로 개발할 예정이므로 다음과 같이 화면에 꽉 차게 설정해야 합니다.
```suggestion
    @supports (width: 100svw) { width: 100svw; }
    @supports not (width: 100svw) { width: 100vw; }
```